### PR TITLE
Rework of the Lyo Sandbox

### PR DESCRIPTION
**Pull request by berezovskyi**
The most of the images are built in 2 stages via makefiles.

Robot simulate uses MQTT to simulate pubsub communication but w/o ROS.

### CODE REVIEW

**Comment by berezovskyi** [comment]
@likelion https://github.com/EricssonResearch/scott-eu/pull/99/commits/d8ef84b01038df357459163a80a3e3e0282cc81f is the relevant commit here

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -11,38 +11,35 @@ stages:
   - deploy
 
 before_script:
-  - apk add --no-cache git
-  - git clone https://github.com/EricssonResearch/scott-eu.git
+  - apk add --no-cache git alpine-sdk
```
Comment (likelion) [inline .gitlab-ci.yml:14]: why do we need alpine-sdk? it slows down the deployment quite a lot.

**Comment by berezovskyi** [comment]
@likelion I have restored placement constraints. shall we merge to master?

**Review by likelion — APPROVED** [review]
Let's try!

File: .gitlab-ci.yml (hunk 1 above)
Comment (berezovskyi) [reply .gitlab-ci.yml:14]: it has gcc & make. we use make

**Pull request merged**